src/api/api.ts

```diff
@@ -45,15 +70,14 @@ export const checkFieldValueExists = async (
   return !querySnapshot.empty;
 };
 
-export const getDocCollection = async (
-  dataBase: Firestore,
-  nameCollection: string,
-  id: string,
-): Promise<DocumentData | void> => {
-  const docRef = doc(dataBase, nameCollection, id);
-  const docInfo = await getDoc(docRef);
+export const getUsers = async (): Promise<QuerySnapshot<DocumentData, DocumentData>> => {
+  const data = await getDocs(usersCollection);
 
-  return docInfo.data();
+  return data;
+};
+
+export const LogOut = async (): Promise<void> => {
+  await signOut(auth);
 };
```

Comment:
doesn't look like a universal method and try adding and add [Order and limit data](https://firebase.google.com/docs/firestore/query-data/order-limit-data?hl=en&authuser=0#order_and_limit_data)